04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -319,17 +331,13 @@ RGB_stack_HARV[[1]]
 hist(RGB_stack_HARV,
      maxpixels=ncell(RGB_stack_HARV))
 
```

Comment:
I removed that code as the lesson has already taught plotting a single band from multi-band rasters. However, I guess the syntax is slightly different so I've added it back in.